example/src/serviceChooser.js

```diff
@@ -1,42 +1,45 @@
 import React from 'react';
-import ShadowDOM from 'react-shadow';
```

Comment:
Let's make sure removing the shadow DOM stuff is OK here. Maybe @WilcoFiers and @dylanb could chime in and suggest whether or not we need it?